Recommend .local/bin instead of /usr/local/bin (#10)
yt-dlp/yt-dlp-wiki: Installation.md

[![MacOS](https://img.shields.io/badge/-MacOS-lightblue.svg?style=for-the-badge&logo=apple)](https://github.com/yt-dlp/yt-dlp/releases/latest/download/yt-dlp_macos)
[![Other variants](https://img.shields.io/badge/-Other-grey.svg?style=for-the-badge)](https://github.com/yt-dlp/yt-dlp#release-files)

> In UNIX-like OSes (MacOS, Linux, BSD), you can also install the application into a location in your path.  For example, you can install into `~/.local/bin` in one of the following ways:

Reviewer: ```suggestion
In UNIX-like OSes (MacOS, Linux, BSD), you can also install the application into a location in your `$PATH`, such as `~/.local/bin`, in one of the following ways:
```
